question-generation/reflectia/src/taskpane.css

```diff
@@ -0,0 +1,118 @@
+/* 
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+html,
+body {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    background: white;
+}
+
+ul {
+    margin: 0;
+    padding: 0;
+}   
+
+.ms-welcome {
+    margin: 10px;
+    display: flex;
+    gap: 20px;
+    flex-direction: column;
+}
+
+.prompt-editor {
+    max-height: 300px;
+}
+
+.cards-container {
+
+}
```

Comment:
This .cards-container seems to be repeating with the .card-container below? Maybe remove this?